code/game/objects/effects/spiderwebs.dm

```diff
@@ -127,19 +127,19 @@
 	AddComponent(/datum/component/swarming)
 
 /obj/structure/spider/spiderling/hunter
```

Comment:
I didn't address this in this PR because it wasn't relevant to anything else but this typepath is fucked up